front-end/src/services/Eosio.js

```diff
@@ -3,14 +3,20 @@ import { JsSignatureProvider } from 'eosjs/dist/eosjs-jssig';
 import ecc from 'eosjs-ecc';
 import { copyObj } from './objects';
 import EosioMyApi from './EosioMyApi';
+import { createDfuseClient } from '@dfuse/client';
 
 import settings from '../settings';
-const defaultNetwork = settings.eosio.network;
 
 class Eosio {
-    constructor(network = defaultNetwork) {
-        let rpc = fetch ? new JsonRpc(network, {fetch}) : new JsonRpc(network);
+    constructor(network = { nodeos: settings.eosio.nodeos, dfuse: ettings.eosio.dfuse }) {
```

Comment:
It seems like we are accessing blockchain using Eosio class and RESTApi through NodeJS. We have to clearly mention why we choose one over another.

Reply:
eosjs is the SDK that uses the HTTP API provided by nodeos (official software provided by Block One) that runs the eosio software

dfuseClient is the SDK that uses the RESTful HTTP API, or the graphql API (including WebSockets) API provided by the Dfuse node that runs the eosio protocol

Dfuse services offer more functionalities:
- ability to flexibly query the blockchain history
- a few extra capabilities to query blockchain state including public key based search (which we need)
- WebSockets so more interactivity (not in use now, but handy to have)
- plus all functionality that comes with nodeos

Additionally, dfuse manages exits more gracefully, so that our management is easier than for nodeos (we don't need to reset the blockchain so often)